packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -19,16 +19,20 @@
 }
 
 // The base width of blocks
-.edit-post-visual-editor .block-editor-block-list__block {
-	margin-left: auto;
-	margin-right: auto;
-
+.edit-post-visual-editor .block-editor-block-list__block.has-spacing {
 	@include break-small() {
 		// Compensate for side UI width.
-		.block-editor-block-list__block-edit {
+		> .block-editor-block-list__block-edit {
```

Comment:
I would encourage us to think of increasing the style specificity as plan C. If there is _anything at all_ we can do to avoid increasing the specificity, it would be much preferable, as this really has compounding effects.